### PR TITLE
Fix/rename admin petition statuses

### PR DESCRIPTION
**Pull request by freesteph**

### CODE REVIEW

File: app/helpers/admin_helper.rb [hunk 1]
```diff
@@ -119,6 +119,23 @@ def back_link
...
+                when 'completed'
+                  'closed'
+                when 'hidden'
+                  'rejected'
+                else
+                  status
+                end
+
+    formatted.humanize
+  end
```
Comment (pixeltrix) [inline app/helpers/admin_helper.rb:137]: The values `referred` and `completed` aren't actual state values so these branches will never be triggered. Probably best to add a `status` method to `Petition`, e.g.

``` ruby
def status
  if closed? && !completed?
    'under_consideration'
  elsif completed?
    'closed'
  elsif open?
    'collecting_signatures'
  elsif hidden?
    'rejected'
  else
    state
  end
end
```

Then you don't need the view helper and you can just change `petition.state.humanize` to `petition.status.humanize`.

**Comment by freesteph** [comment]
[Rename petition states on the Admin site](https://trello.com/c/G4xFURol/151-rename-petition-states-on-the-admin-site)

**Review by pixeltrix — CHANGES_REQUESTED** [review]
Can you also change the bottom left panel on the admin hub screen to 'Under consideration', please.

File: app/models/petition.rb [hunk 2]
```diff
@@ -47,7 +47,7 @@ class Petition < ActiveRecord::Base
 
   facet :all,       -> { not_archived.by_most_recent }
   facet :open,      -> { not_archived.open_state.by_most_popular }
-  facet :rejected,  -> { not_archived.rejected_state.by_most_recent }
+  facet :rejected,  -> { not_archived.rejected_state.or(hidden_state).by_most_recent }
```
Comment (pixeltrix) [inline app/models/petition.rb:50]: I'm going to put my trust in the PostgreSQL query planner here and assume that there's no difference in plan between `state IN ('rejected', 'hidden')` and `state = 'rejected' OR state = 'hidden'` 😂

**Comment by pixeltrix** [comment]
Also change 'Completed' to 'Closed' on the bottom right of the admin hub page.

**Review by pixeltrix — APPROVED** [review]

**Comment by freesteph** [comment]
[As a moderator, when I search for Rejected petitions I want them to show up as "Rejected" rather than "Hidden"](https://trello.com/c/HmhCVOiR/110-as-a-moderator-when-i-search-for-rejected-petitions-i-want-them-to-show-up-as-rejected-rather-than-hidden)

**Pull request merged**